app/src/main/java/com/hedvig/app/util/extensions/DimensionExtensions.kt

```diff
@@ -0,0 +1,6 @@
+package com.hedvig.app.util.extensions
+
+import android.content.res.Resources
+
+val Int.dp: Int
```

Comment:
I don't love the name of this extension `toDp`?

Reply:
I think it's quite a nice name - I stole it from some Jetpack Compose-code that used this type of extension.